Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.5
+ENV PYTHONUNBUFFERED 1
+RUN mkdir /code
+WORKDIR /code
+ADD requirements.txt /code/
+RUN pip install -r requirements.txt
```

Comment:
We could use `python:3.5-onbuild` base image:  
https://github.com/docker-library/python/blob/master/3.5/onbuild/Dockerfile  
```
FROM python:3.5

RUN mkdir -p /usr/src/app
WORKDIR /usr/src/app

ONBUILD COPY requirements.txt /usr/src/app/
ONBUILD RUN pip install --no-cache-dir -r requirements.txt

ONBUILD COPY . /usr/src/app
```  

Note the pip install before the code copy, it reduces the rebuild time.
I also find useful let the default CMD on the Dockerfile, for reference and commodity.